### ISSUE TITLE
[ACCESSIBILITY] Favorites screen - Keyboard trap-  Screen reader user unable to swipe past the image

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#3026]
**Describe the accessibility issue**
As a TalkBack user, I am unable to gesture/swipe past the image to "Good morning Amanda" or any further content. 

**To Reproduce
Steps to reproduce the behavior:**

1. Go to app, login
2. Turn on TalkBack
3. Upon landing on 'Favorites' swipe right to gesture through screen from top to bottom. 

**Expected behavior**
User is able to swipe right hearing the contents of the screen as then navigate from top to bottom of screen. 

**Actual behavior**
User is able to swipe right hearing the contents of the screen only until they reach the image at top. THey are unable to proceed past image. 

**Screenshots**

https://user-images.githubusercontent.com/82892873/224045175-17278aac-7058-49b7-bc47-88f512623a12.mp4

Smartphone (please complete the following information):

•	OS: Android Version 13
•	Device: Google Pixel 4a

Additional context
https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-trapping.html

**Comment by TodorBachvarov** [comment]
I'm not able to reproduce this issue. @acovher2 could you try it again probably it's already fixed.

**Comment by TodorBachvarov** [comment]
Fixed in version v4.3.47

**Comment by acovher2** [comment]
verified as fixed in 4.3.61
